README.md

```diff
@@ -1,8 +1,47 @@
+
+**Core Java Apps** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_HarikrishnaSPillai/tree/master/core_java)]:
+      
+  - JDBC App: Worked on a Java application using JDBC API to connect to a PostgreSQL database. The app allows users to perform CRUD operations and is built using the DAO(Data Access Object) pattern. Maven is used as Project Management Tool.
+  - Grep App: Reimplemented the popular LINUX grep command as a Java App. The application recursively searches through a given directory for a supplied Regex pattern. The matched lines are then written in a text file. The application was improvised by Java 8 features Lambda and Stream API. Streams and Lambda helped in reducing memory usage and inducing a declarative programming approach. IntelliJ was used as IDE and the project was built using Maven. The app is deployed on DockerHub using Docker.
+
+
+## Highlighted Projects
+**Real-Time Parking Detection System**: Designed and Developed a Parking Detection system in a Raspberry Pi and Intel Neural Compute Stick. The system used Yolo v3 Real-time Object detection algorithm to process images supplied by Pi cam. The number of parking spots was conveyed from Raspberry Pi to Firebase using Python and Web API. The data received in real-time was broadcast to the client-side web application. The web app was developed using Flutter and Dart programming language. Additionally, the web app used google authentication API and Map API for authentication and GUI purposes.
+
+**Portfolio Website**: Developed a Responsive website using tech HTML/CSS/JS as front end stack. The back-end was configured using AWS services like S3 and route 53. www.analysthari.com
+
+
+## Professional Experiences
+
+**Software Developer, Jarvis (2020-present)**: Practised Scrum framework for team collaboration and SDLC. Developed several application using technologies such as Docker, Bash scripting, Java, RDBMS(SQL), Project management using Maven and version control using Git/GitHub.
+
+**Product Analyst, BSC Global (2020-2021)**: Delivered quality reports on a biweekly basis using MS excel and Power BI to stakeholders to showcase the improvements made by the department. Prepared an informative dashboard for the logistics department which increased productivity by approximately 10 percent. Improvised the current SOP which increased the productivity by 25 percent with almost similar or decreased human effort.Test and Diagnose products with proper documentation to help organize inventory
```

Comment:
Delivered quality reports on a biweekly basis using MS Excel and Power BI to stakeholders to showcase the improvements made by the department. Prepared an informative dashboard for the logistics department which increased productivity by approximately 10 percent. Improvised the current SOP which increased the productivity by 25 percent with almost similar or decreased human effort. Test and Diagnose products with proper documentation to help organize inventory

Reply:
Changed to MS Excel